modules/cache-material/test/clone_cryptographic_material.test.ts

```diff
@@ -46,7 +46,7 @@ const cryptoKey: any = { type: 'secret', algorithm: { name: webCryptoSuite.encry
 
 describe('cloneMaterial', () => {
   it('clone NodeEncryptionMaterial', () => {
-    const material = new NodeEncryptionMaterial(nodeSuite)
+    const material = new NodeEncryptionMaterial(nodeSuite, { some: 'context' })
```

Comment:
What error is thrown when no context is provided? Should we test for that?

Reply:
You can not create Encryption Materials without encryption context.

https://github.com/awslabs/aws-encryption-sdk-javascript/blob/master/modules/material-management/test/cryptographic_material.test.ts#L430

The public api (encrypt/decrypt) should be the _only_ place that encryption context is optional.